config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -556,7 +556,7 @@ periodics:
       - name: FOCUS
         value: \[sig-network\]|\[Conformance\]
       - name: SKIP
-        value: \[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS|ProxyTerminatingEndpoints)\]|LB.health.check|LoadBalancer|load.balancer|GCE|NetworkPolicy|DualStack
+        value: \[Feature:(Networking-IPv4|Example|Federation|PerformanceDNS|ProxyTerminatingEndpoints)\]|Internet.connection|upstream.nameserver|LB.health.check|LoadBalancer|load.balancer|GCE|NetworkPolicy|DualStack
```

Comment:
See the failing job https://testgrid.k8s.io/sig-network-kind#sig-network-kind,%20IPv6,%20master%20%5Bnon-serial%5D

those tests "IPv4" and "Internet.connection" doesn't work in IPv6

On my previous PR I copied the SKIP line of an IPv4 test by mistake 
https://github.com/kubernetes/test-infra/pull/28029